dnsserv_test.go

```diff
@@ -1,40 +0,0 @@
-package main
-
-import (
-	"bytes"
-	. "gopkg.in/check.v1"
-	"net"
-	"testing"
-)
-
-func Test(t *testing.T) { TestingT(t) }
-
-type DNSNameserverSuite struct{}
-
-var _ = Suite(&DNSNameserverSuite{})
-
-func (s *DNSNameserverSuite) TestUDPConnection(c *C) {
```

Comment:
Eh. The server could listen on any address and port, and I would rather focus on tests that drive out the work needed for the nameserver to respond to queries. #9 is the issue we are working on.